pyuvsim/uvsim.py

```diff
@@ -1010,6 +1010,13 @@ def run_uvsim(input_uv, beam_list, beam_dict=None, catalog_file=None,
     if rank == 0:
         print("Tasks Received. Begin Calculations.")
         stdout.flush()
+
+    for i, bm in enumerate(local_task_list[0].telescope.beam_list):
+        if isinstance(bm, UVBeam):
+            uvb = UVBeam()
+            uvb = bm
+            local_task_list[0].telescope.beam_list[i] = bm
```

Comment:
Maybe add a comment explaining what's happening here and why.

Reply:
Good idea. The cluster seems busy at the moment... the jobs are still pending and the estimated start time isn't for a couple days. In my experience they'll run much sooner than that.